src/RightSidebar/TOC/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import useTOCHighlight from '@theme/hooks/useTOCHighlight';
+
+import styles from './styles.module.css';
+
+const LINK_CLASS_NAME = styles['contentsLink'];
+const ACTIVE_LINK_CLASS_NAME = 'contents__link--active';
+const TOP_OFFSET = 140;
+
+function Headings({headings, isChild}) {
```

Comment:
nit: Spaces after/before curlies in imports and function args/kwargs

Reply:
kwargs?